streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -565,6 +576,7 @@ public void shouldThrowNullPointerExceptionOnRangeNullToKey() {
             bytesValue(value));
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
I'm not sure about this one, do you think we need to use the deprecated method, or can we migrate here?

Reply:
` @SuppressWarnings("deprecation")// continuing to support WindowStore#put(K, V) as internal api, this call uses the 3 params
    @Override
    public synchronized void put(final Bytes key,
                                 final byte[] value) {
        put(key, value, context.timestamp());
    }`
no need to deprecate here